assets/webviews/templates/welcome.ejs

```diff
@@ -11,7 +11,11 @@
 
 <body>
   <div class="section-div">
-    <h2 class="title-header">Quarkus Tools for <%= appName %></h2>
+    <h2 class="title-header">Quarkus Tools for <%= appName %>
+      <% if (status) { %>
+        <span class="preview"><%= status %></span>
+      <% } %>
+      </h2>
```

Comment:
Wrap these in a `<div>` and move the bottom margin from `.title-header` to this new `<div>`. Then, you can use this css to vertically align the preview badge:

```css
.title-wrapper {
  display: flex;
  align-items: center;
  margin-bottom: 12px; /* This is from .title-header */
}
```